example-app/SantokuApp/package.json

```
@@ -16,14 +16,14 @@
     "postprebuild:stg": "run-p copy:stg:*",
     "postprebuild:prod": "run-p copy:prod:*",
     "build:plugin": "rimraf config/plugin/build && tsc --build config/plugin",
-    "copy:local:android": "rimraf prebuild/local/android && cpx \"android/**\" prebuild/local/android",
-    "copy:local:ios": "rimraf prebuild/local/ios && cpx \"ios/**\" prebuild/local/ios",
-    "copy:dev:android": "rimraf prebuild/dev/android && cpx \"android/**\" prebuild/dev/android",
-    "copy:dev:ios": "rimraf prebuild/dev/ios && cpx \"ios/**\" prebuild/dev/ios",
-    "copy:stg:android": "rimraf prebuild/stg/android && cpx \"android/**\" prebuild/stg/android",
-    "copy:stg:ios": "rimraf prebuild/stg/ios && cpx \"ios/**\" prebuild/stg/ios",
-    "copy:prod:android": "rimraf prebuild/prod/android && cpx \"android/**\" prebuild/prod/android",
-    "copy:prod:ios": "rimraf prebuild/prod/ios && cpx \"ios/**\" prebuild/prod/ios",
+    "copy:local:android": "rimraf prebuild/local/android && cpx \"android/**/{.*,*}\" prebuild/local/android",
```

❓ 全く動かしても調べてもないんだけど、これってこんな感じじゃ上手く動かないもんなのかな？

```suggestion
    "copy:local:android": "rimraf prebuild/local/android && cpx android prebuild/local",
```

---

それだと `prebuild/local` が空っぽのままでした。

```diff
     "build:plugin": "rimraf config/plugin/build && tsc --build config/plugin",
-    "copy:local:android": "rimraf prebuild/local/android && cpx \"android/**/{.*,*}\" prebuild/local/android",
-    "copy:local:ios": "rimraf prebuild/local/ios && cpx \"ios/**/{.*,*}\" prebuild/local/ios",
+    "copy:local:android": "rimraf prebuild/local/android && cpx android prebuild/local",
+    "copy:local:ios": "rimraf prebuild/local/ios && cpx ios prebuild/local",
     "copy:dev:android": "rimraf prebuild/dev/android && cpx \"android/**/{.*,*}\" prebuild/dev/android",
```

`cpx` はディレクトリコピー非対応みたいです。

[How to copy dir? · Issue #28 · mysticatea/cpx](https://togithub.com/mysticatea/cpx/issues/28#issuecomment-294787452)

> Currently, no.
I have designed this tool to copy specific files as companion to transpilers. I had not thought copying directories. The feature would be useful.

---

なるほど、ありがとう